core/web/jobs_controller.go

```diff
@@ -110,20 +112,25 @@ func (jc *JobsController) Create(c *gin.Context) {
 	case job.OffchainReporting:
 		jb, err = ocr.ValidatedOracleSpecToml(jc.App.GetChains().EVM, request.TOML)
 		if !config.Dev() && !config.FeatureOffchainReporting() {
-			jsonAPIError(c, http.StatusNotImplemented, errors.New("The Offchain Reporting feature is disabled by configuration"))
+			jsonAPIError(c, http.StatusNotImplemented, errors.New("The Off-chain Reporting feature is disabled by configuration"))
 			return
 		}
 	case job.OffchainReporting2:
 		jb, err = validate.ValidatedOracleSpecToml(jc.App.GetConfig(), request.TOML)
 		if !config.Dev() && !config.FeatureOffchainReporting2() {
-			jsonAPIError(c, http.StatusNotImplemented, errors.New("The Offchain Reporting 2 feature is disabled by configuration"))
+			jsonAPIError(c, http.StatusNotImplemented, errors.New("The Off-chain Reporting 2 feature is disabled by configuration"))
 			return
 		}
 	case job.DirectRequest:
 		jb, err = directrequest.ValidatedDirectRequestSpec(request.TOML)
 	case job.FluxMonitor:
 		jb, err = fluxmonitorv2.ValidatedFluxMonitorSpec(jc.App.GetConfig(), request.TOML)
 	case job.Keeper:
+		// Parse the observationSource here, because it is constant for all the Keeper jobs.
+		_, err := pipeline.Parse(keeper.ExpectedObservationSource)
```

Comment:
I don't think we need to do this for each job creation. Could we parse as part of an `init` block in `keeper/validate.go`

Reply:
Not sure why we need this. Shouldn't the expectedObservationSource be empty?

Reply:
let's rename the variable from 'ExpectedObservationSource' then